RAP/rap-statistics.qmd

```diff
@@ -3,76 +3,84 @@ title: "RAP for Statistics"
+In other areas of analysis, we recommend that RAP principles are applied proportionately. Whilst you wouldn't create a full RAP process for an ad-hoc, you could still version control your code so that it could be reused if similar requests came in, and you should get your code peer reviewed by someone before sending out any results. 
 
----
+Implementing RAP for us will involve combining the use of SQL, R, and clear, consistent version control to increase efficiency and accuracy in our work. For more information on what these tools are, why we are using them, and resources to help upskill in those areas, see our [learning resources](../learning-development/learning-development.html) page.
 
-# Core principles
+The collection of, and routine checking of data as it is coming into the department is also an area that RAP can be applied to. We have kept this out of scope at the moment as the levels of control in this area vary wildly from team to team. If you would like advice and help to automate any particular processes, feel free to [contact the Statistics Development Team](mailto:statistics.development@education.gov.uk).
 
 ---
 
+## Core principles
+
+RAP has three core principles:
 
-**Data sources for a publication are stored in the same database** _- [Preparing data](#preparing-data)_
 
+[Preparing data](#preparing-data): Data sources for a publication are stored in the same database
 
-**Underlying data files are produced using code, with no manual steps** _- [Writing code](#writing-code)_
 
+[Writing code](#writing-code): Underlying data files are produced using code, with no manual steps
 
-**Files and scripts should be appropriately version controlled** _- [Version control](#version-control)_
+
+[Version control](#version-control): Files and scripts should be appropriately version controlled
+
+Within each of these principles are separate elements of RAP. Each of these is discussed in detail below so that you know what is expected of you as an analyst. 
 
  
 ---
 
-# RAP in practice
+## RAP in practice
 
----
 
-The diagram below highlights what RAP means for us, and the varying levels in which it can be applied in all types of analysis. You can click on each of the hexagons in the diagram to learn more about each of the RAP principles and how to use them in practice. The expectation is that all publications will meet the department's baseline implementation of RAP in the self-assessment tool. It's worth acknowledging that some teams are already working around great and best practice levels, and that we appreciate every team's situation is unique, our guidance is designed to be applicable across all official statistics publications by DfE.
+The diagram below highlights what RAP means for us, and the varying levels in which it can be applied in all types of analysis. You can click on each of the hexagons in the diagram to learn more about each of the RAP principles and how to use them in practice. 
```

Comment:
Not sure if just being laggy when running locally, but the 'use appropriate tools' hexagon link doesn't seem to work

Reply:
it had changed - fixed it now!